environment.c

```diff
@@ -69,3 +68,7 @@ void freeTheNodes(struct EnvNode *head)
  * @enVariable: poiner to the environment variable string.
  * Return: returns the value of the environment variable's value.
  */
+char *getEnvironment(char *enVariable)
+{
+	return (getenv(enVariable));
```

Comment:
- I had the idea that getenvironment would after calling the getenv function, tokenize each string of environment variables Add assign variables and value e.g variable `Pat & value` into a node so we have a list of environment variables to be able to use the `setenv` command when resetting variables.